src/newtab/Frame.svelte

```diff
@@ -112,6 +113,34 @@
   //   console.log('lala', $viewingAsset)
   // }
 
+  const dispatch = action => {
+    console.log('dispatch', action);
+    
+    switch (action) {
+      case 'LEFT':
+        onPrev()
+        break
+      case 'RIGHT':
+        onNext()
+        break
+      case 'EXPAND':
+        onExpand()
+        break
+      case 'RANDOM':
+        onRandom()
+        break
+      case 'FIRST':
+        onFirst()
+        break
+      case 'LAST':
+        onLast()
+        break
+      default:
+        console.error('Keyboard shortcut dispatch error. Unhandled error.')
```

Comment:
Small improvement:
```suggestion
        console.error('Keyboard shortcut dispatch error. Unhandled:', action)
```
Then you can see which specifically is unhandled.
Though, I just noticed the log above, so you'd already know. XD